### ISSUE TITLE
rule spaced-comment should report errors when multi-spaces after //

### ISSUE THREAD

**Issue by Aladdin-ADD** [eslint/eslint#9673]
**Tell us about your environment**

* **ESLint Version:**4.12.1

**What parser (default, Babel-ESLint, etc.) are you using?**
default
**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  "rules": {
    "spaced-comment": "error"
  }
}


```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
//     a comments.
//                   another comment.

/**
 * I
 *    am
 *        jsdoc
 */
```

**What did you expect to happen?**
should report an error, since more than one space/tab after the start of a comment `//`

**What actually happened? Please include the actual, raw output from ESLint.**
no error reported. [online demo](https://eslint.org/demo/#eyJ0ZXh0IjoiLyogZXNsaW50IHNwYWNlZC1jb21tZW50OiBcImVycm9yXCIqL1xyXG5cclxuLy8gICAgIGEgY29tbWVudHMuXHJcbi8vICAgICAgICAgICAgICAgICAgIGFub3RoZXIgY29tbWVudC4iLCJvcHRpb25zIjp7InBhcnNlck9wdGlvbnMiOnsiZWNtYVZlcnNpb24iOjgsInNvdXJjZVR5cGUiOiJtb2R1bGUiLCJlY21hRmVhdHVyZXMiOnt9fSwicnVsZXMiOnsic2VtaSI6Miwic3BhY2VkLWNvbW1lbnQiOjJ9LCJlbnYiOnt9fX0=)

**Comment by sakabar** [comment]
@Aladdin-ADD Hi. The [document of spaced-comment](https://github.com/eslint/eslint/blob/v4.12.1/docs/rules/spaced-comment.md) says "If "always" then the // or /* must be followed by **at least one** whitespace". This doesn't mean "the // or /* must be followed by single space", is it?

**Comment by Aladdin-ADD** [comment]
well, seems it's working as intended.

side note: ideally we might need a rule to enforce comments indent for these cases. but it's hard to handle properly, e.g:
```js
// function foo(){
//   return bar;
// }
```
